packages/engine/lib/orchestrator/Cargo.toml

```diff
@@ -20,4 +21,5 @@ uuid = { version = "0.8.1", features = ["v4", "serde"] }
 
 [features]
 default = ["build-nng"]
-build-nng = ["hash_engine_lib/build-nng"]
+build-nng = ["hash_engine_lib/build-nng", "hash_engine/build-nng"]
+texray = ["hash_engine/texray"]
```

Comment:
Are these features necessary now just so they can be forwarded to the dependency?

Reply:
Cargo just looks up all features requested for a specific crate (i.e. `hash_engine_lib`) and combines them. Thus those features are not needed (actually no features are required for the orchestrator). I adjusted it accordingly.